examples/stop_symbol.rs

```diff
@@ -8,6 +8,7 @@ fn main() {
     let mut sp = Spinner::new(
         Spinners::from_str(&spinner_name).unwrap(),
         "Waiting for 3 seconds".into(),
+        None,
```

Comment:
@ad4mx can we keep our backward compatibility and make the parameter color with a default none value? so the upgrade of spinners will be painless?

Reply:
As far as I know, this is impossible in Rust, because you have to supply all needed arguments. I'll look more into it.

Reply:
Looks like merging this PR would require a significant version bump (probably to something like v5.0.0). I can't find a way to keep backwards compatibility in this case.

Reply:
Alright, I think moving it to a dedicated function would be a better idea then. I've also edited the code according to clippy.